docs/CODE_GUIDELINES.md

```diff
@@ -384,14 +415,32 @@ namespace
 
 void test();
 
-}
+} // unnamed namespace
 ```
 
 ❌ Bad:
 ```cpp
 static void test();
 ```
 
+### 6.3. C++ elements
+
+Elements of C++ like functions must always be used by adding the namespace prefix `std`.
```

Comment:
I think this  would be better as "The interface of C++ standard library" instead of a generic "c++ elements". I'd say maybe:

```
When using elements defined in the C+ standard library headers, prefixing the element with the `std` prefix is mandatory.
```